### PR TITLE
Keyboard focus style

### PR DESCRIPTION
**Pull request by AyoobAboosalih**
### Related Issues
- https://github.com/wso2/product-is/issues/15213

### Related PRs
- https://github.com/wso2/identity-apps/pull/3641
- https://github.com/wso2/identity-apps/pull/3642

### Checklist
- [x] e2e cypress tests locally verified.
- [x] Manual test round performed and verified.
- [x] UX/UI review done on the final implementation. - n/a
- [x] Documentation provided. (Add links if there are any) - n/a
- [x] Unit tests provided. (Add links if there are any) - n/a
- [x] Integration tests provided. (Add links if there are any) - n/a

### Security checks
- [x] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [x] Ran FindSecurityBugs plugin and verified report?
- [x] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/authentication-portal/src/main/webapp/emailOtp.jsp [hunk 1]
```diff
@@ -150,7 +150,11 @@
                                 <%
                                     if ("true".equals(authenticationFailed)) {
                                 %>
-                                <a class="ui button secondary"
+                                <a 
+                                    class="ui button secondary" 
+                                    onclick="resendOtp()" 
+                                    tabindex="0" 
+                                    onkeypress="javascript: if (window.event.keyCode === 13) resendOtp()"
```
Comment (AyoobAboosalih) [inline apps/authentication-portal/src/main/webapp/emailOtp.jsp:157]: should line 162 be changed to : (changing authenticate to be wrapped in single quotes)

value="<%=AuthenticationEndpointUtil.i18n(resourceBundle, 'authenticate')%>"

File: apps/authentication-portal/src/main/webapp/smsOtp.jsp [hunk 2]
```diff
@@ -193,12 +198,11 @@
...
-        $(document).ready(function() {
-            $('#resendCodeLinkDiv').click(function() {
-                document.getElementById("resendCode").value = "true";
-                $('#pin_form').submit();
-            });
-        });
+
+        function resendOtp() {
+            document.getElementById("resendCode").value = "true";
+            $("#pin_form").submit();
```
Comment (AyoobAboosalih) [inline apps/authentication-portal/src/main/webapp/smsOtp.jsp:204]: '#pin_form' was changed to -> "#pin_form"

File: apps/authentication-portal/src/main/webapp/smsOtp.jsp [hunk 3]
```diff
@@ -134,13 +134,18 @@
                             </div>
                             <input type="hidden" name="sessionDataKey"
                             value='<%=Encode.forHtmlAttribute(request.getParameter("sessionDataKey"))%>'/><br/>
-                            <div class="align-right buttons">
+                            <div class="align-right buttons" >
```
Comment (brionmario) [inline apps/authentication-portal/src/main/webapp/smsOtp.jsp:137]:
```suggestion
                            <div class="align-right buttons">
```

**Review by brionmario — DISMISSED** [review]

File: apps/authentication-portal/src/main/webapp/smsOtp.jsp (hunk 3 above)
Comment (AyoobAboosalih) [reply apps/authentication-portal/src/main/webapp/smsOtp.jsp]: adjusted with 38e412b7fed713e6bf317a061db2f0c667a4818b

**Review by brionmario — APPROVED** [review]

**Pull request merged**